tests/integration/mininet_tests.py

```diff
@@ -7688,6 +7681,7 @@ def setUp(self): # pylint: disable=invalid-name
 
     def test_untagged(self):
         """Stack loop prevention works and hosts can ping each other."""
+        self._set_event_log(self.EVENT_LOGGER_TIMEOUT)
```

Comment:
seems like EVENT_LOGGER_TIMEOUT should be pulled up, and then just made a default in the enable function?